CMakeLists.txt

```diff
@@ -36,15 +34,15 @@ find_package(catkin REQUIRED COMPONENTS
 )
 
 catkin_package(
-  INCLUDE_DIRS include
+  INCLUDE_DIRS
+    include
   LIBRARIES ${PROJECT_NAME} ${OCTOMAP_LIBRARIES}
   CATKIN_DEPENDS
     eigen_stl_containers
     random_numbers
     shape_msgs
     visualization_msgs
   DEPENDS
-    Eigen
```

Comment:
This must not be removed, but has to be replaced by `EIGEN3` in this request, just as in https://github.com/v4hn/geometric_shapes/commit/5493cb4079024bd44ce09f8404c2e4e797713193#diff-af3b638bc2a3e6c650974192a53c7291R46 .
We provide the header `geometric_shapes/bodies.h` that directly includes some eigen3 headers, so we have to depend on the package here.